app/views/assignments/_read.html.erb

```diff
@@ -33,11 +37,13 @@
 
   <% unless @assignment.submission_rule.type == "NoLateSubmissionRule" || @grouping&.extension&.apply_penalty == false %>
     <h3><%= SubmissionRule.model_name.human.capitalize %></h3>
-    <p><%= @assignment.submission_rule.class.human_attribute_name(:description) %></p>
+    <p><%= @assignment.submission_rule.class.human_attribute_name(:description) unless @assignment.is_timed %></p>
 
     <ul>
       <% if @grouping.nil? %>
         <li><%= t('grace_period_submission_rules.no_group_yet') %></li>
+      <% elsif @assignment.is_timed? %>
+        <li><%= t('assignments.timed.full_duration_warning_html') %></li>
```

Comment:
This warning is no longer relevant (there's no more notion of a "final due date"). I think we *do* want to show the "due date with penalty" (in the else block), but only `unless @assignment.is_timed? && @grouping.start_time.nil?`.